recipes/pyopcode/bld.bat

```diff
@@ -0,0 +1,29 @@
+set BUILD_CONFIG=Release
+REM tell cmake where Python is
+set PYTHON_LIBRARY=%PREFIX%\libs\python%PY_VER:~0,1%%PY_VER:~2,1%.lib
+
+REM work in build subdir
+cd pyopcode
+mkdir build
+cd build
+
+cmake ../src -G%CMAKE_GENERATOR% ^
+    -Wno-dev ^
```

Comment:
Strange, because a colleague used a version of this bld.bat file to build it for 2.7 where the flag was there. I've reverted back to using a manually constructed GENERATOR_NAME to see if that does the trick